CarbonFootprint/GetterSetter/ClassWithAutoImpProperty.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GetterSetter
+{
+    public class ClassWithAutoImpProperty
+    {
```

Comment:
no need to mention class word in class name

Reply:
updated